04_verify.sh

```diff
@@ -221,7 +227,7 @@ RESULT_STR="Fetch CRDs"
 CRDS="$(kubectl --kubeconfig "${KUBECONFIG}" get crds)"
 process_status $? "Fetch CRDs"
 
-if [ "${CAPI_VERSION}" == "v1alpha2" ]; then
+if [[ "${CAPI_VERSION}" == "v1alpha2" ]] || [[ "${CAPI_VERSION}" == "v1alpha3" ]]; then
```

Comment:
why the "or" here ?

Reply:
my current assumption is that EXPTD_V1ALPHA2_CRDS are valid for v1a3 too.